prep-example | Daniel To

theDanielTo opened the pull request:
![prep-example](https://user-images.githubusercontent.com/82009814/115124569-c3cf4f00-9f77-11eb-89bd-1b6ccbe74bf2.png)

theDanielTo committed:
Completed prep-example.

thebearingedge reviewed (changes requested):
Hi Daniel! Your HTML is empty for some reason! 😟

thebearingedge commented:
https://github.com/theDanielTo/prep-0421-code-solutions/pull/1/files

theDanielTo committed:
Deleted example.html

theDanielTo commented:
> Hi Daniel! Your HTML is empty for some reason! 😟

I accidentally committed the wrong change to prep-example. The example.html file was added as a test file.  I've deleted that (example.html) file and updated the prep-example branch.

Here's the link to my prep-example.html code:
https://github.com/theDanielTo/prep-0421-code-solutions/blob/master/prep-example/index.html
